public/index.html

```diff
@@ -52,19 +52,19 @@ <h4><span class="step">(Step 1 / 6)</span></h4>
 	            				<h4>Doctor's Details <span class="step">(Step 2 / 6)</span></h4>
 	            				<div class="form-group">
 				                    <label>Doctor's Full Name:</label><br>
-				                    <input type="text" name="doctor_name" class="form-control" id="doctor_name" required>
+				                    <input type="text" name="doctor_name" placeholder="Full Name" class="form-control" id="doctor_name" required>
```

Comment:
Please make all the doctor details readonly. Also add specialization of doctor in doctor details. Don't make this specialization readonly. Doctor should be able to edit the specialization.

Reply:
Done.